### PR TITLE
refactor(server/app/serve_index.go): replaced literal 302 with http constant - #1004

### LINKED ISSUE #1004: Replace numerical literal 302 with constant http.StatusFound in server/app/serve_index.go
Instead of using a numerical status code, we should use the constant declared in net/http.

### Old
Screenshot from 2021-04-08 21-17-54

### New
Screenshot from 2021-04-08 21-20-25

### PR DESCRIPTION
**Pull request by AeronCyther**
# Closes 
#1004 

# Changes 
Replaced a numerical literal with a constant from net/http

### CODE REVIEW

File: server/app/serve_index.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func serveIndex(ds model.DataStore, fs fs.FS) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		base := path.Join(conf.Server.BaseURL, consts.URLPathUI)
 		if r.URL.Path == base {
-			http.Redirect(w, r, base+"/", 302)
```
Comment (jvoisin) [inline server/app/serve_index.go:25]: A `302` isn't a `200` :/

Comment (deluan) [reply server/app/serve_index.go:25]: This is correct. GoLang weirdly names 302 as `http.StatusFound`. Ref: https://golang.org/pkg/net/http/

Comment (AeronCyther) [reply server/app/serve_index.go:25]: It was originally 302, so I just replaced it with the appropriate constant. Should I do 200 instead?

Comment (jvoisin) [reply server/app/serve_index.go:25]: Nevermind, I thought that http.StatusFound was a 200, but it's indeed a [302](https://golang.org/pkg/net/http/)

**Review by jvoisin — APPROVED** [review]

**Review by deluan — APPROVED** [review]
Thanks!

**Pull request merged**